daemon/mgr/container.go

```diff
@@ -386,6 +386,32 @@ func (mgr *ContainerManager) Create(ctx context.Context, name string, config *ty
 		return nil, err
 	}
 
+	// check disk quota
+	// TODO: regexp check and one quota for rootfs and all of volumes.
+	if meta.Config.DiskQuota == nil {
```

Comment:
To be honest, this part of code takes me quite a long time to understand what you design in this diskquota API.
Here is my initial understanding:

* User can configure a container's rootfs with two ways: directly flag and labels with prefix `DiskQuota` which is case-sensitive;
* If user have not set diskquota in flag, we should extract diskquota related data in labels. To be honest, I think code here should be `if len(meta.Config.DiskQuota) == 0` rather than 0-comparing;
* Id user sets diskquota in labels but also sets in flag, we will ignore the data in labels, right?

This is your design, If what I understand is correct.
Not to mention the correctness of it, I am wondering if we could discuss the design first and make it clear before we start to code. Idea sharing first will reduce the code reviewer's time a lot on guessing what the code writer's purpose. Also it will be easily to express oneself clearly with explicit design and architecture, then I am afraid the document the feature should be mapped to would be quite easy to write. WDYT?

I do think design first and design review is an urgent work we should do, otherwise communication overhead and redoing work will take quite much energy of our team.